### DISCUSSION TITLE
Mini-w1-2ch

### DISCUSSION THREAD

**Discussion by fisht99** [arendst/Tasmota#18958] [General]
Hi There

Anyone ever flashed one of these with Tasmota?
https://www.ebay.com.au/itm/125984860775?hash=item1d55486a67:g:gMEAAOSwgmBkkry~

Haven't purchased any yet but cant seem to find any info on them.

![image](https://github.com/arendst/Tasmota/assets/25100632/b3653234-d14b-4f04-8bf8-d4c2d878aa3e)

**Comment by sfromis** [comment]
As a starting point, my assumption for such devices would be that they are not using a "Tasmota-compatible" MCU chip, unless having reason to believe otherwise.

**Reply by fisht99** [reply]
I have ordered one. When it arrives i will take a look and report back.

**Reply by sfromis** [reply]
By checking the MAC in pairing mode, and looking it up on the net, you can find out without cracking it open.

**Reply by fisht99** [reply]
Switch arrived today, comes up on Tuya as a PSW-3CH-CB2S

Not sure if I can be bothered going through the transplant process!

**Reply by cweemin** [reply]
I got the same (2 channel instead). Have you figured if it's compatible?

**Reply by barbudor** [reply]
I have purchased 2 of them to check, choosing models claiming Ewelink as this generally increase the chance of being ESP based (vs Tuya that totally gave up ESP) 

Unfortunately it is fitted with Buffalo modules, not compatible
Just received my ESP-02S modules for brain transplant

The exact one I purchased : 
https://a.aliexpress.com/_EG23xaZ

1st transplant OK
Template: `{"NAME":"Mini Smart Switch","GPIO":[1,1,1,1,320,160,1,1,224,32,1,1,1,1],"FLAG":0,"BASE":18}`

**Reply by sfromis** [reply]
In general, you can check the MAC in pairing mode via some "Wifi Analyzer" app, and look it up on a site like `macvendors.com`. If it is not "Espressif", no go for Tasmota.

**Reply by max-men-cat** [reply]
> I have purchased 2 of them to check, choosing models claiming Ewelink as this generally increase the chance of being ESP based (vs Tuya that totally gave up ESP) 
> 
> 
> 
> Unfortunately it is fitted with Buffalo modules, not compatible
> 
> Just received my ESP-02S modules for brain transplant
> 
> 
> 
> The exact one I purchased : 
> 
> https://a.aliexpress.com/_EG23xaZ
> 
> 
> 
> 1st transplant OK
> 
> Template: `{"NAME":"Mini Smart Switch","GPIO":[1,1,1,1,320,160,1,1,224,32,1,1,1,1],"FLAG":0,"BASE":18}`
> 
> 

hello. Is it possible to share the template for both channels? with this template only one relay works

**Reply by sfromis** [reply]
You can just try setting the 2nd relay pin in "Configure Template" until you find where it works.
